### PR TITLE
Add Matplotlib Tutorial Video

### PR DESCRIPTION
**Pull request by Kacawi**

### CODE REVIEW

File: datacamp/category.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "description": "Learn-by-doing in the comfort of your own browser. DataCamp teaches you Python for data science at your own pace! Interactive coding challenges and fun quizzes related to the videos can be found on `DataCamp's Website <https://www.datacamp.com/>`__.",
```
Comment (logston) [inline datacamp/category.json]: Thanks you for your contribution! However, our philosophy of keeping everything here free somewhat conflicts with advertising courses on our site. Can you update your description to something less promotional? Thank you.

File: datacamp/videos/Matplotlib-Plot-Tutorial-For-Beginners.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+  "copyright_text": "Standard YouTube License",
+  "description": "Matplotlib makes it easy to create meaningful and insightful plots. In this beginner video you will learn how to build various types of data visualizations such as histograms, scatter plots and line plots. You will also see how to customize them to make them more visually appealing and interpretable. \n\nWant to do the corresponding exercises? Go to our `Python For Data Science Tutorial <https://www.datacamp.com/courses/intro-to-python-for-data-science>` where you can do them for free.",
```
Comment (logston) [inline datacamp/videos/Matplotlib-Plot-Tutorial-For-Beginners.json]: Same comment as above.

File: datacamp/videos/Matplotlib-Plot-Tutorial-For-Beginners.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "duration": 600,
+  "language": "eng",
+  "recorded": "2016-02-01",
+  "related_urls": [
+    "https://www.datacamp.com/courses/intermediate-python-for-data-science"
```
Comment (logston) [inline datacamp/videos/Matplotlib-Plot-Tutorial-For-Beginners.json]: Same comment as above.

**Comment by logston** [comment]
Thanks @Kacawi. Please collapse this branch into a single commit and I'd be happy to merge.

**Comment by Kacawi** [comment]
Thanks! 👍

**Pull request merged**